packages/jaeger-ui/src/components/DeepDependencies/Header/LayoutSettings/index.test.js

```diff
@@ -66,15 +68,6 @@ describe('LayoutSettings', () => {
     expect(trackDensityChangeSpy).toHaveBeenCalledWith(props.density, newDensity, densityOptions);
   });
 
-  it('no-ops if current density is selected', () => {
-    getWrapper()
-      .find(Radio)
-      .at(densityIdx)
-      .simulate('change', { target: { value: props.density } });
-    expect(props.setDensity).not.toHaveBeenCalled();
-    expect(trackDensityChangeSpy).not.toHaveBeenCalled();
-  });
```

Comment:
why did you remove this test?

Reply:
I thought because of the new local storage addition for setting a default configuration, this test would be trivial and not necessary because the default configuration will be updated every time

Reply:
This test is not about defaults, but about what happens when the current selection is re-selected.

Reply:
Update: I have rewrittend the tests, which makes sure trackDensityChange is not being called, and it is working fine

Reply:
![Screenshot 2023-12-20 122945](https://github.com/jaegertracing/jaeger-ui/assets/103821431/2cd26e51-84bc-4d4c-93f2-c9d997754ec0)
All tests have successfully passed and new tests added and old tests restored.

Reply:
@yurishkuro I have handled the test case when current selection is reselected.
Should I raise a PR now?

Reply:
which PR?